packages/ui/src/composables/useMaxSelections.ts

```diff
@@ -23,9 +23,8 @@ export function useMaxSelections (
     return selections.value.length >= maxSelections.value
   }
 
-  const addOption = (optionToAdd: any): void => {
-    const newSelectedOptions = [...selections.value, optionToAdd]
-    emit('update:modelValue', newSelectedOptions)
```

Comment:
Also would be great to remove `emit` from the function arguments. :) And also remove emits where it was passed.